analysislib/known_dns_sigs.py

```diff
@@ -753,6 +753,238 @@
     """
     ),
 
+    SimpleDNSReplyPattern(
+        name="dns_isp_ru_AS52207",
```

Comment:
I wasn't sure what naming scheme you were using and since I automatically generated the fingerprint blobs I went for something that was fairly straightforward and easy (i.e. just using the ASN as an identifier).